scss/_variables.scss

```diff
@@ -851,7 +832,6 @@ $progress-height:               1rem !default;
 $progress-font-size:            .75rem !default;
 $progress-bg:                   $gray-lighter !default;
 $progress-border-radius:        $border-radius !default;
-$progress-box-shadow:           inset 0 .1rem .1rem rgba($black,.1) !default;
```

Comment:
This one can remain; I'll need to add a conditional shadow to the `.progress` to make use of it.